### DISCUSSION TITLE
Add conditions to the respawn locations

### DISCUSSION THREAD

**Discussion by mat-ha3** [Zrips/CMI#6908] [Enhancement]
![image](https://user-images.githubusercontent.com/67873769/160458758-3db4bd44-77cf-41f8-8f6b-51825a33ddf4.png)
Like in the image above. ^^^
Being able to add conditions, like eventcommands, to respawn locations would be good

**Comment by CloudeLecaw** [comment]
Event command cinditions?
You allready can prioize them by world
And give diverent spawns to different groups.

You know you have a section event for respawn in eventCommands.yml for commands that happen on respawn?  And if u need it earlier u have one for death.
The prority here just is fornwhat respawns should take first if given.
If u have more commands and you neee to work in a order then use soecialzed commands delay inneventCommamd to proize an Order of your commands.
